openmetadata-service/src/main/java/org/openmetadata/service/resources/databases/DatabaseResource.java

```diff
@@ -76,6 +77,7 @@ public Database addHref(UriInfo uriInfo, Database db) {
     return db;
   }
 
+  @ResourceConstructor(type = ResourceConstructor.ConstructorType.WITH_DAO_AUTH)
```

Comment:
@sureshms Collection on constructor feels a bit out of place, since it's already used for marking resource classes and i would have to do something like (Collection.ConstructorType.WITH_DAO_AUTH.toString()) , which has less type enforcing :) , let me know wdyt , will update it!! :)

Reply:
@mohityadav766 I don't understand what out of place means.

You don't need `ResourceConstructor` and you can `ConstructorType` enum in `Collection` annotation itself with a field `authorization`. That is the class that is meant for initializing a resource. Let me know what you think...

Reply:
updated with the review comments